pkg/printers/internalversion/describe.go

```diff
@@ -3794,23 +3794,16 @@ func printTaintsMultilineWithIndent(w PrefixWriter, initialIndent, title, innerI
 	}
 
 	// to print taints in the sorted order
-	keys := make([]string, 0, len(taints))
-	for _, taint := range taints {
-		keys = append(keys, string(taint.Effect)+","+taint.Key)
-	}
-	sort.Strings(keys)
+	sort.Slice(taints, func(i, j int) bool {
+		return string(taints[i].Effect)+","+taints[i].Key < string(taints[j].Effect)+","+taints[j].Key
```

Comment:
Can you turn this 'stringify' method into a helper to ensure both are actually the same.